src/app.js

```diff
@@ -29,34 +35,42 @@ export default (
   ): mixed => {
     if (request.method === 'OPTIONS') {
       response.set({
-        'Access-Control-Allow-Headers':
-          'X-Requested-With, Content-Type, Accept, Authorization',
+        'Access-Control-Allow-Headers': 'X-Requested-With, Content-Type, Accept, Authorization',
         'Access-Control-Allow-Methods': 'GET, POST, PUT, DELETE, OPTIONS',
         'Access-Control-Allow-Origin': '*',
         'Access-Control-Max-Age': '300',
       });
       return response.sendStatus(204);
     }
-    response.set({ 'Access-Control-Allow-Origin': '*' });
+    response.set({
+      'Access-Control-Allow-Origin': '*',
+    });
     return next();
   };
 
   if (settings.LOG_REQUESTS) {
-    app.use(morgan(
-      '[:date[iso]] :remote-addr - :remote-user ":method :url ' +
-      'HTTP/:http-version" :status :res[content-length] ":referrer" ' +
-      '":user-agent"',
-    ));
+    logger.warn('Request logging enabled');
+    if (!logger.debug()) logger.warn('Request will not log, cause Bunyan loglevel is different!');
+    const useLogger = logger;
+    app.use(bunyanMiddleware({
```

Comment:
This is fine for now but I want to make sure we still get a lot of info from the request logging